pkg/controller/kubedirectorcluster/members.go

```diff
@@ -284,7 +284,21 @@ func handleReadyMembers(
 				return
 			}
 
-			memberVersion := *m.StateDetail.LastConnectionVersion
+			var memberVersion int64
+
+			if m.StateDetail.LastConnectionVersion != nil {
+				memberVersion = *m.StateDetail.LastConnectionVersion
+
+			} else {
+				memberVersion = connectionsVersion - 1
```

Comment:
Can you tell me a bit more about why we default to "connectionsVersion - 1" if there's no lastConnectionVersion?

Reply:
Yes, I'm going to add comments and then convert this to a PR that is ready for review

Reply:
👍

Reply:
Added comments in the PR description and code. Feel free to take a look.